src/main/resources/static/codegen/built-api-spec.yml

```diff
@@ -6066,6 +6066,38 @@ components:
           example: "M1502750438"
       required:
         - crn
+    Cas2NewApplication:
+      type: object
+      properties:
+        crn:
+          type: string
+        nomsNumber:
+          type: string
+        name:
+          type: string
+        dateOfBirth:
+          type: string
+          format: date
+        nationality:
+          type: string
+        sex:
+          type: string
+        personStatus:
+          type: string
+          enum:
+            - InCustody
+            - InCommunity
+            - Unknown
+        prisonName:
+          type: string
+        pncNumber:
```

Comment:
This might be different for your users but PNC was certainly a very important number in cas3 reports.

Reply:
It's come up as something Nacro ask for but not sure how required it is for a referral - it's an optional field on offenders returned from the probation-offender-search so we have to account for some cases where it might be missing.